xdsl/transforms/convert_memref_stream_to_snitch_stream.py

```diff
@@ -232,16 +172,29 @@ def strides_for_affine_map(
     """
     if affine_map.num_symbols:
         raise ValueError("Cannot create strides for affine map with symbols")
-    offset_map = strides_map_from_memref_type(memref_type)
+    offset_map = memref_type.get_affine_map()
+    if offset_map.num_symbols:
+        # only allowed for dynamic offsets, since we can ignore this here
+        # in that case, there would be one symbol, and all shapes are static
```

Comment:
I'm not sure exactly what this means

Reply:
I tried changing the check for clarity. We don't want any symbols for dynamic strides, as this is not supported yet. However, a dynamic offset is supported, as the offset can be ignored in this function. I just changed the check to make sure all shapes are statically known. If there are any symbols left in the affine map, they are related to the offset, and can just be set to 0.